test/bandit/http2/plug_test.exs

```diff
@@ -241,6 +241,20 @@ defmodule HTTP2PlugTest do
     assert response.body == "BCD"
   end
 
+  test "sending a chunked file", context do
+    {:ok, response} =
+      Finch.build(:get, context[:base] <> "/chunked_file_test")
+      |> Finch.request(context[:finch_name])
+
+    assert response.status == 200
+    assert response.body == File.read!(Path.join([__DIR__, "../../support/sendfile_chunked"]))
+  end
+
+  def chunked_file_test(conn) do
+    conn
+    |> send_file(200, Path.join([__DIR__, "../../support/sendfile_chunked"]), 0, :all)
+  end
+
```

Comment:
I think the actual intent of this test is to ensure that it does the right thing with 'large' files. Suggest changing the language here to be something like 'sending a file which is larger than 2048 bytes'. I mention this mostly just because 'chunk' has a specific meaning within HTTP and the fact that we use chunking internally is really just an implementation detail that's not really the business of the test. Ultimately we just want to ensure that we get the file read back to us.

Reply:
I updated the naming to make the intention of the test case clearer.